plugins/fritz-box7490/de/fritzbox7490_

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+# Munin plugin um die aktuellen Werte der Datei fritzbox7490.log auszuwerten. 
+# internet connection by reading the top_status.htm from the 
+# Don't forget zu fill in the following lines into the munin-node
+#
+# - normally at /etc/muni/plugin-conf.d/ - an than restart munin
+#
+#   [fritzbox_]
+#   user root
+#
+# LLOSS: Leitungsdämpfung   = line loss
+# DRATE: Datenrate          = data rate
+# SNR:   Störabstand        = signal-to-noise ratio
+# DATAV: Datenvolumen       = data transfer
+# ES:    Sek. m. Fehlern    = sec. w. errors
+# SES:   Sek. m. v. Fehlern = sec w. m. errors
+# VERB:  Anzahl tägl. Verb. = count of daily connections
+# UPT:   Zt. seit Reboot    = router uptime
+#                                     Jo Hartmann (Version 16.0126)
+#
+
+###################################
+## Personal config Section Begin ##
+################################### 
+
+logfile=/var/log/fritzbox7490.log
+
+###################################
+## Personal config section End   ##
+###################################
+
+function Echo() {
```

Comment:
This function name decreases readability. Please use a more descriptive and less confusing (echo != Echo) name.